torch/nn/_intrinsic/qat/modules/conv_fused.py

```diff
@@ -251,3 +259,11 @@ def forward(self, input):
     @classmethod
     def from_float(cls, mod, qconfig=None):
         return super(ConvReLU2d, cls).from_float(mod, qconfig)
+
+def update_bn_stats(mod):
+    if type(mod) in set([ConvBnReLU2d, ConvBn2d]):
```

Comment:
you can also just do 'hasattr' (in case more modules appear in the future)

Reply:
Is this safer? We want to modify this field only for specific modules. Tagging based on a specific attribute could be theoretically more risky as we would set freeze_bn to false for any module that has this field. Leaving this as is so that we have explicit control over which modules we apply this modification to.